src/System.Management.Automation/utils/Telemetry.cs

```diff
@@ -620,6 +630,11 @@ static ApplicationInsightsTelemetry()
                     };
 
                 s_uniqueUserIdentifier = GetUniqueIdentifier().ToString();
+                s_knownSubsystemNames = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+                    {
+                        "Completion",
+                        "general"
```

Comment:
How about adding [Az.Tools.Predictor](https://www.powershellgallery.com/packages/Az.Tools.Predictor/1.1.3) and [Microsoft.WinGet.CommandNotFound](https://www.powershellgallery.com/packages/Microsoft.WinGet.CommandNotFound/1.0.4.0) to the list?

Reply:
Sure, we can definitely add to this list (it might also want to be others), i just don't know what they might be

Reply:
@daxian-dbw Do you know the _names_ of these two? it's the `Name` property of the implementation of IFeedbackProvider

Reply:
I believe winget is `Windows Package Manager - WinGet` and `Az Predictor` for Az predictor.

Reply:
i've added those